File: Java/Searching Algorithms/LinearSearch.java
public class LinearSearch {

    public static int search(int[] arr, int key) {
        int n = arr.length;
        for (int i = 0; i < n; i++) {
            if (arr[i] == key)
                return i;
        }
        return -1;
    }

    public static void main(String[] args) {
        int[] arr = {2, 3, 4, 10, 40};
        int searchKey = 10;

        int result = search(arr, searchKey);
        System.out.print(result == -1 ? "Element is not present in array" : "Element is present at index " + result);
    }
}

File: javascript/linearSearch.js
function LinearSearch(arr, ele) {
    for (var i = 0; i < arr.length; i++) {
      if(arr[i] == ele) {
          return console.log(i);
      }
    }
    return console.log('Element not found')
  }
  
  var arr = [234, 43, 55, 63, 5, 6, 235, 547]
  
  LinearSearch(arr, 5)
  